### PR TITLE
added time feature

### PR DESCRIPTION
**Pull request by Shah-Arsalan**
Added a real time clock feature
Screenshot (278)

### CODE REVIEW

**Review by Aman-Jain1996 — APPROVED** [review]
Ui looks great , nice use of variables , good to merge

**Review by PeeyushPrashant — APPROVED** [review]
clean and modular code

File: src/MainPage/MainPage.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Clock } from "../Components";
+import "./MainPage.css";
+
+const MainPage = () => {
+  return (
+    <>
```
Comment (vinc3nati) [inline src/MainPage/MainPage.jsx:6]: Try omitting fragments wherever not necessary.

**Review by vinc3nati — APPROVED** [review]
Clean UI and modular code

**Pull request merged**